lib/utils.js

```diff
@@ -928,6 +929,14 @@ function makeClientMetadata(options) {
 
 const noop = () => {};
 
+function emitDeprecatedOptionWarning(options, name, list) {
+  list.forEach(option => {
+    if (options[option]) {
```

Comment:
this should be `if (typeof options[option] !== 'undefined')`, otherwise you'll get some false positives

Reply:
updated!